add_gem_package.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -e
 
 usage() {
-	echo "Usage: $0 GEM_NAME [TEMPLATE [PACKAGE_SUBDIR [KOJI_TAG]]]"
+	echo "Usage: $0 GEM_NAME [TEMPLATE [PACKAGE_SUBDIR [REPO]]]"
 	echo "Valid templates: $(ls gem2rpm | sed 's/.spec.erb//' | tr '\n' ' ')"
-	./list_koji_tags
+	echo "Valid repos: foreman-el8 foreman-plugins-el8 katello-el8 foreman-client"
```

Comment:
is this missing the `el8` from client intentionally?

Reply:
Client has many distros, and the logic appears to ignore that part anyway.

Reply:
Fair, but in L68 you write:
```
if [[ $REPO == "foreman-client-el8" ]] ; then
```

and the `add_gem_to_comps $REPO` in L64 only works for "valid" repos that match to comps files 1:1.

;-)

Reply:
> Fair, but in L68 you write:

I like to call it editing.